source/helpers/FormatCompletions.ts

```diff
@@ -0,0 +1,23 @@
+interface CompletionsType {
+  approved: boolean;
+  description: string;
+}
+const getUnapprovedCompletionDescriptions = (
+  completions: CompletionsType[]
+): string[] => {
+  const completionDescriptions = completions.reduce(
+    (previous: string[], current: CompletionsType) => {
+      const unapprovedCompletion = !current?.approved;
+
+      if (unapprovedCompletion) {
+        return [...previous, current.description];
+      }
+      return previous;
+    },
+    []
+  );
```

Comment:
A more concise approch is 
`completions.filter(({approved}) => approved).map(({description}) => description)`

Reply:
I agree. We should use filter and map instead of reduce here, it makes the code more readable.